website/docs/externaldata.md

```diff
@@ -430,3 +430,104 @@ server := &http.Server{
 ```
 
 2. If `cert-controller` is disabled via the `--disable-cert-rotation` flag, you can use a cluster-wide, well-known CA certificate for Gatekeeper so that your external data provider can trust it without being deployed to the `gatekeeper-system` namespace.
+
+### Authenticate API server against Webhook (Self managed kube cluster only)
+
+To have requests coming from apiserver authenticated by webhook, following configuration can be made:
+
+1. Deploy gatekeeper with cert rotator disabled and with a client ca name. Provide name of the client ca with the flag `--client-cert-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate apiserver requests if both - cert rotator disabled and client ca name is provided with flag - of the criteria are met.
```

Comment:
Is there a reason cert rotation needs to be disabled for this to work? Is there a way to make this orthogonal? Maybe store the cert in a separate secret/configmap (IIUI  WRT secret vs. configmap the cert is not sensitive)

Reply:
Interesting. Why can't we use the same secret? does cert rotator delete and creates new secret while rotating cert or does it just update the whole data field making it impossible to persist manually added entries to data map?

Reply:
TBH I can't remember how cert-rotator behaves, but I wouldn't be surprised if it clobbered the whole data field.

In any case, if we could use the same secret I'd still like to enable/disable either option separately for operational flexibility.